src/commcare_cloud/ansible/roles/pgbouncer/handlers/main.yml

```diff
@@ -10,3 +10,15 @@
 - name: Reload pgbouncer
   become: yes
   action: service name=pgbouncer state=reloaded
+
+- name: Start pgbouncer2
+  become: yes
+  action: service name=pgbouncer2 state=started
+
+- name: Restart pgbouncer2
+  become: yes
+  action: service name=pgbouncer state=restarted
```

Comment:
Should this say `Restart pgbouncer`? Or name=pgbouncer2? Or is there something else I'm missing

Reply:
name should be `pgbouncer2` good catch